README.md

```diff
@@ -0,0 +1,85 @@
+
+
+#### Table of Contents
+
+1. [Description](#description)
+2. [Setup - The basics of getting started with at](#setup)
+    * [What at affects](#what-at-affects)
+3. [Usage - Configuration options and additional functionality](#usage)
+4. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+5. [Limitations - OS compatibility, etc.](#limitations)
+6. [Development - Guide for contributing to the module](#development)
+    * [Acceptance Tests - Beaker env variables](#acceptance-tests)
+
+
+## Description
+
+This module manages the at service and /etc/at.allow.
+
+
+### This is a SIMP module
+
+This module is a component of the [System Integrity Management Platform](https://github.com/NationalSecurityAgency/SIMP), a compliance-management framework built on Puppet.
+
+If you find any issues, they can be submitted to our [JIRA](https://simp-project.atlassian.net/).
+
+This module is optimally designed for use within a larger SIMP ecosystem, but it can be used independently:
+
+ * When included within the SIMP ecosystem, security compliance settings will be managed from the Puppet server.
+
+
+## Setup
+
+
+### What at affects
+
+  * atd service
+  * `/etc/at.deny`
+  * `/etc/at.allow`
+
+## Usage
+
+To use this module, just call the class. This example adds it to a class list in hiera:
+
+```yaml
+---
+classes:
+  - at
+```
+
+Users can also be added to `/etc/at.allow` with the `at::user` defined type, or
+the `at::users` array in hiera. The following example adds a few users to `/etc/at.allow`:
+
+```yaml
+at::users:
+  - foo
+  - bar
+```
+
+
+## Reference
+
+Please refer to the inline documentation within each source file, or to the module's generated YARD documentation for reference material.
+
+
+## Limitations
+
+SIMP Puppet modules are generally intended for use on Red Hat Enterprise Linux and compatible distributions, such as CentOS. Please see the [`metadata.json` file](./metadata.json) for the most up-to-date list of supported operating systems, Puppet versions, and module dependencies.
+
+
+## Development
+
+Please read our [Contribution Guide] (http://simp-doc.readthedocs.io/en/stable/contributors_guide/index.html).
+
+
+### Acceptance tests
+
+This module includes [Beaker](https://github.com/puppetlabs/beaker) acceptance tests using the SIMP [Beaker Helpers](https://github.com/simp/rubygem-simp-beaker-helpers).  By default the tests use [Vagrant](https://www.vagrantup.com/) with [VirtualBox](https://www.virtualbox.org) as a back-end; Vagrant and VirtualBox must both be installed to run these tests without modification. To execute the tests run the following:
+
+```shell
+bundle install
+bundle exec rake beaker:suites
```

Comment:
This module only has spec tests.  Should the test instructions be for running these spec tests.